administration/src/bp-modules/applications/ApplicationCard.tsx

```diff
@@ -113,14 +114,35 @@ const RightElement = ({ jsonField, application }: RightElementProps): ReactEleme
     return !!blueCardJuleicaEntitlement
   }
 
+  const isPreVerified = (): boolean => {
+    const applicationDetails = findValue(jsonField, 'applicationDetails', 'Array') ?? jsonField
+    const workAtOrganizationsEntitlement =
+      findValue(applicationDetails, 'blueCardWorkAtOrganizationsEntitlement', 'Array')?.value ?? []
+
+    const isAlreadyVerified = workAtOrganizationsEntitlement.some(
+      (entitlement: GeneralJsonField) =>
+        Array.isArray(entitlement.value) &&
+        entitlement.value.some(
+          (organizationField: GeneralJsonField) =>
+            organizationField.name === 'isAlreadyVerified' && organizationField.value === true
+        )
+    )
+    return isAlreadyVerified
+  }
+
+  let quickIndicator
```

Comment:
:wrench:
I think redeclaring variable with let should be avoid as far as possible and i think the redundant declaration of the `<PreVerifiedQuickIndicator/>`  react component could be avoided by first checking if `isPreVerified()` even only a different type will be passed to the same component

sth. like this look cleaner to me
```js
const isPreVerified = isJuleicaEntitlementType() || isPreVerifiedByOrganization()

  return (
    <RightElementContainer>
      {!!application.note && application.note.trim() && <Icon icon='annotation' intent='none' />}
      {isPreVerified ? (
        <PreVerifiedQuickIndicator
          type={
            isJuleicaEntitlementType() ? PreVerifiedQuickIndicatorType.Juleica : PreVerifiedQuickIndicatorType.Verein360
          }
        />
      ) : (
        <VerificationsQuickIndicator verifications={application.verifications} />
      )}
    </RightElementContainer>
  )
```

the function above could be renamed to `isVerifiedByOrganization` to make it clearer. Since juleica is also preVerified but not by organization